src/components/ModalAmount/ModalAmount.tsx

```diff
@@ -47,40 +56,28 @@ export const Modal = (props: Props) => {
       </button>
 
       <h2>{props.sellerName}</h2>
-      <p>Please select an amount and leave a message</p>
+      <p>Please select an amount or enter any amount</p>
 
       <div className={styles.amountContainer}>
         <label htmlFor="select-amount">Select an amount </label>
         <br />
         <div className={styles.selectAmtContainer}>
-          <button
-            type="button"
-            className={'modalButton--outlined'}
-            onClick={(e) => handleAmount(10, false)}
-          >
-            $10
-          </button>
-          <button
-            type="button"
-            className={'modalButton--outlined'}
-            onClick={(e) => handleAmount(25, false)}
-          >
-            $25
-          </button>
-          <button
-            type="button"
-            className={'modalButton--outlined'}
-            onClick={(e) => handleAmount(50, false)}
-          >
-            $50
-          </button>
-          <button
-            type="button"
-            className={'modalButton--outlined'}
-            onClick={(e) => handleAmount(100, false)}
-          >
-            $100
-          </button>
+          {buttonAmounts.map((amount) => (
```

Comment:
great approach, maybe create a separate handler for the amount submission for writing cleaner tsx/jsx